src/pandantic/basemodel.py

```diff
@@ -1,132 +1,27 @@
 """A subclass of the Pydantic BaseModel that adds a parse_df method to validate DataFrames."""
+
 from __future__ import annotations
```

Comment:
Not a comment on an addition, but why is this line here? My linter says it is used, but i don't see where. More a curiosity question.

Reply:
What are you using for linting?

Reply:
I use `pyright`, if it looks fine on `mypy` or whatever you are using I say this doesn't matter. Btw what is this for?